server/exec_functests

```diff
@@ -12,5 +12,8 @@ source /var/tmp/venv/bin/activate
 python3 -m pip install . -r client/requirements.txt
 
 PBENCH_SERVER=${1} python3 -m pytest ${PWD}/lib/pbench/test/functional/server
+rc=$?
```

Comment:
Our normal style would call for `${?}`

Reply:
A quick survey suggests that we have 162 files which refer to `$?` 278 times, whereas `${?}` is used 340 times in 68 files...I would call that a split decision.  😁

I prefer omitting the braces when they are not necessary, particularly for the single-character special variable names...but, since we did it at line 14, I s'pose we can do it here, too.  🥴